documentation for TextInput (#8605)
slint-ui/slint: docs/astro/src/content/docs/reference/keyboard-input/textinput.mdx

import SlintProperty  from '/src/components/SlintProperty.astro';
import Link from '/src/components/Link.astro';

The `TextInput` is a lower-level item that shows text and allows entering text.
You should probably not use this directly, but instead use the <Link type="LineEdit" /> or <Link type="TextEdit" /> component.

> When not part of a layout, its width or height defaults to 100% of the parent element when not specified.

Reviewer: ```suggestion
When not part of a layout, its width and height defaults to 100% of the parent element.
```
